src/Microsoft.Extensions.DataIngestion/Processors/SummaryEnricher.cs

```diff
@@ -45,7 +45,7 @@ public async Task<List<DocumentChunk>> ProcessAsync(List<DocumentChunk> chunks,
             [
                 new(ChatRole.User,
                 [
-                    new TextContent($"Write a summary text for this text with less than {_maxWordCount} words."),
+                    new TextContent($"Write a summary text for this text with less than {_maxWordCount} words. Return just the summary."),
```

Comment:
Ollama is not as smart as Open AI. It was always prefixing the answer with sth like: "Here is the Summary:"